pkg/authn/k8schain/tests/serviceaccount/main.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+
+	"github.com/google/go-containerregistry/pkg/authn/k8schain"
+)
+
+func main() {
+	ref, err := name.NewTag("gcr.io/build-crd-testing/secret-sauce:latest", name.WeakValidation)
+	if err != nil {
+		log.Fatalf("NewTag() = %v", err)
+	}
+
+	kc, err := k8schain.NewInCluster(k8schain.Options{
+		Namespace:          "serviceaccount-namespace",
+		ServiceAccountName: "serviceaccount",
+		// ImagePullSecrets: []string{
```

Comment:
Intentionally left?

Reply:
Ah, I think this simply indicates the name of the pull secret attached to the service account.  I added a clarifying comment, but it is intentionally commented out.